modules/va_forms/app/models/va_forms/form.rb

```diff
@@ -18,12 +18,12 @@ class Form < ApplicationRecord
     validates :title, presence: true
     validates :form_name, presence: true
     validates :row_id, uniqueness: true
-    validates :url, presence: true
-    validates :sha256, presence: true
```

Comment:
These fields don't technically need to be present, for example, if the form URL is not valid.